### PR TITLE
Introduction To Network Scanning Using Nmap

### LINKED ISSUE #1263: Introduction to Network Scanner-Nmap
### Brief Summary: 
 Nmap is a network scanning tool that uses IP packets to identify all the devices connected to a network and to provide information on the services and operating systems they are running.
Whether you are a private user with important information on your system, a major corporation or a government agency protecting a wealth of highly sensitive data, Nmap can provide the level of knowledge and pre-emptive thought required to keep things safe.

### Key Takeaways: 
1.Performing Port Scanning.
2.Operating System and Version Detection .
3.Finding live hosts on a network.
### References: 
N/A

### PR DESCRIPTION
**Pull request by jumamiller**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x]  Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x]  Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x]  Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x]  Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
-[x]  Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x]  Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x]  Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x]  In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x]  Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x]  Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x]  Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
This fixes #1263

### CODE REVIEW

File: articles/nmap-network-scanner/index.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+Hit:4 http://ke.archive.ubuntu.com/ubuntu focal-backports InRelease            
+Hit:5 http://ppa.launchpad.net/micahflee/ppa/ubuntu focal InRelease            
+Hit:6 https://download.docker.com/linux/ubuntu focal InRelease                 
+Hit:7 http://security.ubuntu.com/ubuntu focal-security InRelease
+Reading package lists... Done
+jumamiller@OpijaKaeli:~$ 
+
+```
+
+#### Step 2: Install Network Mapper(Nmap)
```
Comment (lalith1403) [inline articles/nmap-network-scanner/index.md:108]: Ensure spaces before parentheses. Install Network Mapper (Nmap). I observed it throughout your article. Change it wherever applicable.

File: articles/nmap-network-scanner/index.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+### SYN Stealth Scan
+
+In simple terms, SYN scan is used by the penetration tester without establishing a full connection, hence the name half-open scanning.  
+
+For example, you want to scan a remote server. You will establish a TCP/IP connection with a server at every possible open port by sending an SYN packet(synchronization packet) as if to create a 3-way handshake to every single port.    
+
+If the synchronization is acknowledged, the server responds with SYN/ACK packet from a particular port, then that particular port is open.  
+
+From that point, you can send a reset packet(RST); consequently, the server assumes that an error occurred while establishing a connection while the open port remains open and ready/vulnerable to exploitation.
```
Comment (lalith1403) [inline articles/nmap-network-scanner/index.md:257]: space after reset packet.

File: articles/nmap-network-scanner/index.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+### SYN Stealth Scan
+
+In simple terms, SYN scan is used by the penetration tester without establishing a full connection, hence the name half-open scanning.  
+
+For example, you want to scan a remote server. You will establish a TCP/IP connection with a server at every possible open port by sending an SYN packet(synchronization packet) as if to create a 3-way handshake to every single port.    
+
+If the synchronization is acknowledged, the server responds with SYN/ACK packet from a particular port, then that particular port is open.  
+
+From that point, you can send a reset packet(RST); consequently, the server assumes that an error occurred while establishing a connection while the open port remains open and ready/vulnerable to exploitation.  
+The client may send several SYN packets to the server, consuming a lot of resources. This way, legitimate requests cannot be handled. This is one-way of how DOS(Denial of service is achieved).
```
Comment (lalith1403) [inline articles/nmap-network-scanner/index.md:258]: space after DOS.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written article. Congratulations! Make the few requested changes and we can proceed to the next round of review,

**Comment by jumamiller** [comment]
@lalith1403, done!

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
just an FYI @jumamiller  The 3rd plagiarism check had a few red flags but I believe most of them were needed dependancies in the code. 
But just an FYI.

**Comment by jumamiller** [comment]
@hectorkambow noted with thanks!

**Pull request merged**